go/database/mpt/forest_test.go

```diff
@@ -2155,6 +2155,54 @@ func TestForest_VisitPathToStorage(t *testing.T) {
 	}
 }
 
+func TestForest_HasEmptyStorage(t *testing.T) {
+	for _, variant := range variants {
+		for _, config := range allMptConfigs {
+			for forestConfigName, forestConfig := range forestConfigs {
+				t.Run(fmt.Sprintf("%s-%s-%s", variant.name, config.Name, forestConfigName), func(t *testing.T) {
```

Comment:
I think it should also test situations where a storage is deleted either by setting an empty value or calling `ClearStorage`

Reply:
and when deleting the whole account

Reply:
When you set an empty AccountInfo: https://github.com/Fantom-foundation/Carmen/blob/main/go/database/mpt/nodes.go#L1546